opennms-model/src/main/java/org/opennms/netmgt/model/OnmsAlarm.java

```diff
@@ -1113,6 +1113,7 @@ public void unacknowledge(String ackUser) {
     @Override
     public void clear(String ackUser) {
         m_severity = OnmsSeverity.CLEARED;
+        acknowledge(ackUser);
```

Comment:
This solves the reported problem - but I don't think we want clears to result in automatic acknowledgements.

Reply:
So having reverted the code pressing clear on the alarm screen leaves the ackTime and ackUser fields blank on the alarms table. However there is a row placed in the acks table, and when we look at the cleared alarm in the UI it shows that an acknowledgment has taken place.

The acknowledge method just populates the ackTime and ackUser fields. Shouldn't we populate these two fields to maintain consistency?

![image](https://user-images.githubusercontent.com/104017065/173842953-f668a284-6db6-4149-a458-7c5c58455028.png)

![image](https://user-images.githubusercontent.com/104017065/173842856-b4f6c1cc-686a-4e78-9794-281c2496e6d2.png)

![image](https://user-images.githubusercontent.com/104017065/173842751-432aac2a-f6da-486b-899a-7acc567e387f.png)

<img width="478" alt="image" src="https://user-images.githubusercontent.com/104017065/173843230-e79c9086-e4d3-430d-a7c2-88296207c4ba.png">

Reply:
I think we don't need to alter the behavior of cleared alarms.  
The issue is more about whether we can route UI page to all alarms if alarm already got deleted.

I favor current behavior which indicates that alarm got deleted and I don't think we need any changes here.
Pinging @jeffgdotorg  for his opinion.